docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -201,6 +201,7 @@ mvn package -Pnative -Dnative-image.docker-build=true
 
 The produced executable will be a 64 bit Linux executable, so depending on your operating system it may no longer be runnable.
 However, it's not an issue as we are going to copy it to a Docker container.
+Note that in this case the build itself runs in a Docker container, too, so you don't need to have GraalVM installed locally.
```

Comment:
`Docker container, too` perhaps should be `Docker container too`?

Reply:
I think so, too :P

Reply:
Pun intended I guess :P

Reply:
Ah, those subtleties always so tough for us non-native speakers :) Just [double checked](https://www.grammarly.com/blog/comma-before-too/), the comma there *may* be added. I'll remove it, though, as there seems common dislike of it :)

Reply:
Comma removed.

Reply:
Are we sure about this sentence? I believe the plugin check for GRAALVM_HOME. (`QuarkusNative:213`)

Reply:
Seems like that code doesn't run in that case; at least I don't have GraalVM installed and can build a container image with a native binary via `mvn clean install -Pnative -Dnative-image.docker-build=true`. Wheres an error is raised when omitting the -D property:

> Failed to generate a native image: GRAALVM_HOME was not set

Reply:
Great! So there is a switch somewhere ;-)